modules/billing-dashboard/main.tf

```diff
@@ -0,0 +1,230 @@
+terraform {
+  required_providers {
+    lightstep = {
+      source  = "lightstep/lightstep"
+      version = "~> 1.60.2"
+    }
+  }
+  required_version = ">= v1.0.11"
+}
+
```

Comment:
This resource block includes one chart which repeats the same metric with the same attributes/settings in 13 query blocks. That can't be correct.

Reply:
@nslaughter that can be manipulated by user but will create an issue to handle a particular metrics suffix pattern for CW Metrics streaming , if there are duplicate metric names.
Ex.:
For every  metric name, CW assess suffixes as shown:
`
*_count
*_max
*_min
*_sum
`
`
aws.billing.estimated_charges_count
aws.billing.estimated_charges_max
aws.billing.estimated_charges_min
aws.billing.estimated_charges_sum
`